R/tags.R

```diff
@@ -1,6 +1,19 @@
 #' @import utils digest
 NULL
 
+# Like base::paste, but converts all string args to UTF-8 first.
+paste8 <- function(..., sep = " ", collapse = NULL) {
+  args <- c(
+    lapply(list(...), enc2utf8),
+    list(
+      sep = if (is.null(sep)) sep else enc2utf8(sep),
+      collapse = if (is.null(collapse)) collapse else enc2utf8(collapse)
+    )
+  )
+
+  do.call("paste", args)
```

Comment:
I'd recommend not to quote `paste`, otherwise it can be unpredictable where this `paste` function could come from. If you do not quote it, R knows it is from the base package.